src/pages/FindFriends.jsx

```diff
@@ -29,22 +29,29 @@ const FindFriends = () => {
           <div className="text-2xl capitalize font-[600]">find friends</div>
         </div>
         <SearchInput setSearchValue={setSearchValue} />
-        <article className="shadow-md p-4   rounded-md space-y-5">
+        <article className="p-4  rounded-[25px] space-y-5 shadow-lg">
```

Comment:
Same here. Define this value `25px` in the tailwind config.

Reply:
done sir